model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -553,11 +514,9 @@ message IntDistributionData {
   int64 max = 4;
 }
 
-message DoubleDistributionData {
-  int64 count = 1;
-  double sum = 2;
-  double min = 3;
-  double max = 4;
+message IntGaugeData {
```

Comment:
Is there a use case for this at the moment? Are you implementing these right now?